.github/workflows/codecov_ats.yml

```diff
@@ -139,8 +142,10 @@ jobs:
     # Something like pytest <options> $ATS_TESTS_TO_RUN
     debug:
       runs-on: ubuntu-latest
-      needs: coverage-ats
-      if: ${{ always() }}
+      needs:
+        - coverage-ats
+        - files-changed
+      if: needs.files-changed.outputs.backend == 'true'
```

Comment:
This `if` makes sure that for `frontend` PRs this workflow will look something like the image.

Example run from https://github.com/giovanni-guidini/sentry/actions/runs/6653707375/job/18081116014?pr=18

![Screenshot 2023-10-26 at 09 59 09](https://github.com/getsentry/sentry/assets/99758426/83081d6c-16e2-443d-86d3-7a07f7ca3197)